ARGUS.py

```diff
@@ -410,15 +410,16 @@ def start_scraping():
     byte_size = int(e13.get())*1000000    # convert from MB to B
     settings_txt.write(settings_file.format(e1.get(), e2.get(), e3.get(), e4.get(), e5.get(), e6.get(), e8.get(), e9.get(), e10.get(), e11.get(), byte_size, e14.get(), e15.get()))
     settings_txt.close()
-    scrapyd_txt = open(script_dir + r"/scrapyd.conf", "w", encoding="utf-8")
```

Comment:
Warum das? Hat das vorher denn überhaupt funktioniert?

Reply:
Funktioniert hat es, war aber meines Erachtens falsch, hat also einfach ein zweites File in einem anderen Ordner erstellt

Reply:
Also vorher lag die Datei weiter oben und du hast sie einfach zusätzlich woanders abgelegt?

Reply:
Ne, als ich die relativen Pfade geändert hab, hab ich das wohl übersehen oder falsch gemacht. Ich überprüfe es aber nochmal

Reply:
Ja, stimmt so